projects/extension/sql/idempotent/005-chunking.sql

```diff
@@ -2,8 +2,7 @@
 -------------------------------------------------------------------------------
 -- chunking_character_text_splitter
 create or replace function ai.chunking_character_text_splitter
-( chunk_column pg_catalog.name
-, chunk_size pg_catalog.int4 default 800
+( chunk_size pg_catalog.int4 default 800
```

Comment:
Should we keep versions of these functions around with the old signature which throw an error telling the user to use the new API?

Reply:
thanks, as it has been decided to move forward instead, marking this as resolved.